assets/source/setup-guide/app/style.scss

```
@@ -345,40 +300,18 @@ $root: ".woocommerce-setup-guide";
 
 				.components-card__body {
 
-					> *:not(:last-child) {
-						margin-bottom: 16px;
-					}
-				}
-
-				#{$root} {
-
-					&__step {
-
-						&-status {
-							padding: 0;
-						}
+					> *:not(:first-child) {
```

💅  `*` is not necessary. 

```suggestion
					> :not(:last-child) {
```

---

Thanks! And I found that it should still select `:not(:last-child)`, I updated your code suggestion to fix it by the way.